fastlane/lib/fastlane/actions/download_dsyms.rb

```diff
@@ -113,8 +113,12 @@ def self.run(params)
               break
             end
 
-            self.download(download_url, app.bundle_id, train.version_string, build.build_version, output_directory)
-            break if build_number
+            if download_url
```

Comment:
I guess this check should be included inside the loop, or the option to wait the symbols will have no effect.

Reply:
This change does indeed break the whole feature of #15049 since the script now no longer waits for the dsym url to appear and just continues as before
@jhandguy FYI